src/components/index.ts

```diff
@@ -58,3 +58,4 @@ export {useOnFocusOutside} from './utils/useOnFocusOutside';
 export * from './utils/interactions';
 export * from './utils/xpath';
 export {getLayersCount} from './utils/LayerManager';
+export {getUniqId} from './utils/common';
```

Comment:
Exporting `useUniqId` hook is not enough?

Reply:
No, as one may use class components